package.json

```diff
@@ -167,6 +168,9 @@
   "resolutions": {
     "@types/react": "16.9.42",
     "@types/react-dom": "16.9.10",
+    "@typescript-eslint/eslint-plugin": "4.22.0",
+    "@typescript-eslint/experimental-utils": "4.22.0",
+    "@typescript-eslint/parser": "4.22.0",
```

Comment:
I forced these resolutions because `eslint-plugin-deprecation`, the web-components eslint config, and maybe a couple other things still use older versions of the deps. It's not ideal but appears to be working okay at the moment.

Reply:
regarding that deprecation plugin, this is the reason why? https://github.com/gund/eslint-plugin-deprecation/blob/master/package.json#L27